en/docs/remote-apps/connect-remote-app.md

```diff
@@ -0,0 +1,120 @@
+# Connect Remote Applications
+
+If you need to observe an external application that you have written in Ballerina programming language, you can connect it to Choreo.
+
+## Prerequisites
+
+Create an external application in Ballerina programming language and run it. For instructions, see [Ballerina Documentation - Writing Your First Ballerina Program](https://ballerina.io/learn/user-guide/getting-started/writing-your-first-ballerina-program/).
```

Comment:
Under per-requisites, i think we dont need to mention to `run` the Ballerina program. With the step 3 we mention how to build and run.
Having a Ballerina project would be enough for the user to connect to Choreo

Reply:
Done